src/pages/publication/MyPublications.tsx

```diff
@@ -11,22 +17,36 @@ export interface DummyPublicationListElement {
   status: string;
 }
 
+const StyledSection = styled.section`
+  text-align: center;
+`;
+
 const MyPublications: FC = () => {
   const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const [isLoading, setIsLoading] = useState(true);
+  const [publications, setPublications] = useState<DummyPublicationListElement[]>([]);
```

Comment:
kunne godt godt brukt skikkelig navn for typen men en gang imo. Vi som må komme med krav til endpoint uansett, så vi kan bare legge inn det vi trenger

Reply:
var usikker på om vi bare skulle bruke den typen vi hadde allerede, eller lage en egen type til liste-bruk